### PR TITLE
Update LINUX.setup.rst

### PR DESCRIPTION
**Pull request by eayoungs**
This file path has changed in the project and I found the directions confusing

### CODE REVIEW

File: docs/source/LINUX.setup.rst [hunk 1]
```diff
@@ -96,7 +96,7 @@ install ``npm`` assuming the prerequisites are met.
 Django Database Configuration
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Create a ``local_untracked.py`` file in the ``BE/settings`` directory and add
+Copy the ``local_untracked.py.dist`` file in the ``config/settings`` directory, remove the .DIST affix and add
```
Comment (nllong) [inline docs/source/LINUX.setup.rst]: thanks for catching this!

**Comment by nllong** [comment]
@eayoungs can you make this PR base develop? Changes are fine.

**Comment by eayoungs** [comment]
I'm not sure how to change the target branch. Do I have to submit a new PR?

**Comment by mmclark** [comment]
Yes, that would be the easiest way to do it.  Just close this PR and 
open a new one against the develop branch.

Thanks

**Comment by RDmitchell** [comment]
I think other places have been identified that need fixing. So maybe close this PR and we will straighten this and a few more things out in the next week or so. Thanks

**Pull request closed without merging**